shell/platform/android/surface/android_surface_mock.cc

```diff
@@ -18,7 +18,8 @@ bool AndroidSurfaceMock::GLContextClearCurrent() {
   return true;
 }
 
-bool AndroidSurfaceMock::GLContextPresent(uint32_t fbo_id) {
+bool AndroidSurfaceMock::GLContextPresent(uint32_t fbo_id,
+                                          std::vector<SkIRect> damage) {
```

Comment:
Is there a test somewhere that verifies that the SkIRect we were expecting was delivered to this call?